nixos/modules/services/x11/display-managers/sddm.nix

```diff
@@ -33,6 +33,8 @@ let
 
       # Implementation is done via pkgs/applications/display-managers/sddm/sddm-default-session.patch
       DefaultSession = optionalString (dmcfg.defaultSession != null) "${dmcfg.defaultSession}.desktop";
+
+      DisplayServer = if cfg.wayland.enable then "wayland" else "x11";
```

Comment:
This makes it unnecessarily hard to use the x11-user greeter.

Reply:
No? It'll just merge your settings on top.

Reply:
Right, I missed the proper context of this.